tests/test_rich_markup_mode.py

```diff
@@ -38,3 +41,304 @@ def main(arg: str):
 
     result = runner.invoke(app, ["--help"])
     assert any(c in result.stdout for c in rounded)
+
+
+@pytest.mark.parametrize(
+    "mode,lines",
+    [
+        pytest.param(
+            "markdown",
+            ["First line", "", "Line 1", "", "Line 2", "", "Line 3", ""],
+            marks=pytest.mark.xfail,
+        ),
+        pytest.param(
+            "rich",
+            ["First line", "", "Line 1", "", "Line 2", "", "Line 3", ""],
+            marks=pytest.mark.xfail,
+        ),
```

Comment:
This one fails on `master` only because of a missing `""` right after the header